ROADMAP.md

```diff
@@ -0,0 +1,226 @@
+# Docs WG Roadmap
+
+## Jan 2016 → Apr 2016 (Q1)
+
+This is the first of the quarterly roadmaps that this Working Group will
+produce, so it will be structured a bit differently than subsequent ones. Since
+this is a brand-new process, an explanation of **what it is** and **why it's
+necessary** follows.
+
+This roadmap outlines the priorities for the Docs working group for the
+quarter. These tasks can be as broadly or narrowly defined as is useful. They
+are not necessarily staffed — as folks step up to do the work, we'll link
+tracking issues to the tasks for them to self-organize. Every quarter we'll
+revisit this roadmap, adding a quick postmortem of how things went, and
+creating a new set of priorities for the ensuing quarter.
+
+That brings us to why a roadmap is necessary: the Node.js documentation is a
+relatively confined space to work within, and there are many interested
+parties: there are groups that consume the documentation as output, like the
+[website working group][wg-website], there are groups concerned with ensuring
+the [content addresses the needs of diverse audiences][wg-inclusivity], there are
+[groups that are required][wg-ctc] to produce API documentation as a
+pre-requisite to merging their primary work, there are contributors who wish to
+help in a _technical_ fashion, there are contributors who wish to contribute
+_editorially_, and there are the readers of the docs. All of these
+contributions overlap, so in order to make measurable progress without stepping
+on each other's feet, we have to:
+
+* **Set clear priorities** — if two contributions conflict, it should be
+  straightforward to determine which contribution should take precedence.
+* **Message these priorities well in advance** — with this many stakeholders,
+  we should mention repeatedly, loudly, and clearly what our intentions are.
+  The Docs WG should aspire to avoid surprise.
+* **Make the best use of donated time** — we want to ensure that if someone is
+  donating their time to improve the documentation, it will be well-spent, and
+  directed at driving the documentation towards this WG's [stated goals][goals].
+
+[Our goals][goals] can be summarized thusly:
+
+> Node.js should be in friendly competition for "Best Docs in OSS," with docs
+> that address the needs of a wide variety of audiences — across skill levels,
+> goals, and languages.
```

Comment:
I have no big stake in it, but I think not having some features as e.g. the es-switcher thing nodejs/node#4915 could work against this. Having a look at newer project's docs they are all the same and imo lack e.g. interactiveness. Sorry for bringing that up. Couldn't read up on the wg meeting yet and also couldn't participate.